.github/workflows/test.yml

```diff
@@ -40,8 +50,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run tests and code coverage analysis
-        run: make -C build coverage
+      - name: Run tests without code coverage analysis
```

Comment:
So it seems that Tarantool 3.2 coverage still segfaults? What's about running coverage with 2.11?